app/ingest_raw/Dockerfile-ingest_raw

```diff
@@ -0,0 +1,5 @@
+FROM python:3
+MAINTAINER James Lamb <jaylamb20@gmail.com>
+
+# TODO (jaylamb20@gmail.com):
+# Implement raw data ingestion service to write to S3
```

Comment:
if we want to automate the process of scraping data regularly, we would need this. Until then, we can just run the scripts I created ad-hoc and get data into s3. But, I agree that we will have to implement kind of scheduled jobs to get the latest JDs & Salary info into s3.

Reply:
yeah this PR isn't actually implementing this service. It's just defining that it's a thing that needs to exist in our flow. The end product of this project should be an application you can pick up and run...at the end of this project, we should not have anything left that says "run this ad hoc script"

Reply:
you're right that it's totally appropriate for us to be doing this ad hoc right now...but that doesn't mean that having an ingestion service is unnecessary in the grander vision